package.json

```diff
@@ -71,7 +71,7 @@
     "danger": "^6.0.5",
     "gulp": "^4.0.2",
     "lerna": "^3.15.0",
-    "lage": "^0.19.0",
+    "lage": "^0.19.8",
```

Comment:
just curious, why are you bumping this?

Reply:
I've been trying out `lage` locally and I wanted support for the `--to` flag (Ken said it was added more recently). Probably going to switch our local build commands to use it soon too.